src/Elliptic/Executables/Poisson/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBS_TO_LINK
```

Comment:
The indentation in this file is inconsistent with a lot of our CMake code. We've been doing 2 space indentation in them. I realize this probably came from another file, but we should try moving to being more consistent on this :)